hack/get_shas.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VERSIONS="7.1.4 8.0.10 9.0.0-M7"
```

Comment:
This script probably isn't necessary anymore. The CI should handle this going forward.